packages/cli/lib/index.ts

```diff
@@ -36,30 +42,35 @@ const program = new NangoCommand();
 dotenv.config();
 
 program.name('nango').description(
-    `By default, the CLI assumes that you are using Nango Cloud so you need to set the NANGO_SECRET_KEY env variable or pass in the --secret-key flag with each command.
+    `By default, the CLI assumes that you are using Nango Cloud so you need to set the NANGO_SECRET_KEY env variable.
 
 For Self-Hosting: set the NANGO_HOSTPORT env variable or pass in the --host flag with each command.
 
-Global flags: --secret-key, --host, --auto-confirm, --debug (output verbose logs for debugging purposes)
+Global flags: --auto-confirm, --debug (output verbose logs for debugging purposes)
```

Comment:
Should we officialise `--debug`? I feel the behavior is not intuitive and too impactful, since the debug behavior is "deployed". I feel it could cause some confusion. Maybe we just use it in 1:1 support conversations for now?

Reply:
Agree, updated with https://github.com/NangoHQ/nango/pull/760/commits/c3fbb79ba1d6c812997931e2492358b3ea58507d